### DISCUSSION TITLE
Möglichkeit um Regelung kurzfristig auszuschalten

### DISCUSSION THREAD

**Discussion by crisi-solar** [tbnobody/OpenDTU#2004] [Ideas]
Hallo!
Wäre es möglich einen Menüpunkt einzubauen wo ich die Regelung z.B. von 8 bis 17 Uhr alle 30 Minuten für 30 Sekunden ausschalten kann?
Der Hintergrund ist der da ich durch die Drosselung nicht weiß welche Maximalleistung die Module in der Lage wären zu leisten um z.B. einen größeren Verbraucher einzuschalten.
Derzeit behelfe ich mich mit einem 2000W-Wasserkocher den ich alle 30 Minuten für 30 Sekunden von 10 bis 16 Uhr einschalte. Bei HomeAssistant habe ich Verbraucher hinterlegt die sich erst bei einer Solarleistung von über 1000W einschalten sollen. Wenn aber die Grundlast z.B. über openDTU auf 400W geregelt wird weiß ich ja nie wann die Solarleistung über 1000W ist wenn ich das nicht mit dem Wasserkoche überprüfe.
Ich hoffe es ist verständlich erklärt.

**Comment by nakott** [comment]
Hallo,
Du könntest doch über die Web API /api/limit/config das Limit selber zeitgesteuert anpassen. 
https://www.opendtu.solar/firmware/web_api/
Siehe auch hier: https://github.com/tbnobody/OpenDTU/discussions/602
Ich selber nutze HomeAssistant nicht, nehme aber stark an, dass man damit so etwas automatisieren könnte. 
Viele Grüße Dirk

**Reply by crisi-solar** [reply]
Danke für die Infos!
Ich muss leider gestehen dass ich mit den Links nichts anzufangen weiß, ist für mich komplettes Neuland.
Da ich zwei WR habe und die Regelung für die Nulleinspeisung entweder über einen SMS oder Raspy mit HoymilesZeroExport mache muss wahrscheinlich die Unterbrechung der Regelung auf einem dieser Geräte passieren. 
Richtig?

**Reply by MetaChuh** [reply]
@crisi-solar frag mal @reserve85 auf seiner projekt seite [`HoymilesZeroExport`](https://github.com/reserve85/HoymilesZeroExport/issues)

die opendtu firmware selbst bleibt vorerst die schnittstelle zu den invertern, mit dem langzeit erprobten status, dass alles so stabil und sorglos wie möglich funktionieren soll.

für zusatzfunktionen, die nicht die mehrheit betreffen, gibt es add on und partner projekte, sodass diese nicht im base projekt für alle eingebaut werden müssen.

thx & greetings,
metachuh

**Comment by nakott** [comment]
mit dem [HoymilesZeroExport](https://github.com/reserve85/HoymilesZeroExport) hast Du ja schon eine Regelung, die kontinuierlich das Limit im Homyles anpasst. Ein neuer Knopf im GUI der OpenDTU würde Dir dann auch eher nichts nützen, denn er käme sich ja in die Quere mot HoymilesZeroExport. 
In diesem Link https://github.com/tbnobody/OpenDTU/discussions/602 ist ein Python Script. Vielleicht kannst Du erst HoymilesZeroExport stoppen, dann mti dem Python Script das Limit auf 100% setzen und dann nach 30 Sekunden HoymilesZeroExport wieder starten. Im Detail kann ich Dir dazu aber nicht mehr sagen.

**Comment by crisi-solar** [comment]
Danke für die Infos!
Wie ich sehe ist die derzeitige Lösung mit dem Wasserkocher und Tasmotastecker für mich die einfachste.
